lib/cext/ruby/ruby.h

```diff
@@ -1502,7 +1502,7 @@ rb_ulong2num_inline(unsigned long v)
     if (RB_POSFIXABLE(v))
 	return LONG2FIX(v);
     else
-	return rb_uint2big(v);
+	return truffle_invoke(RUBY_CEXT, "rb_ulong2num", (long) v);
```

Comment:
I can't see where rb_ulong2num is defined, did you mean `ULONG2NUM`?

Reply:
@bjfish This was your suggestion. Any ideas?

Reply:
```
$ jt ruby -e 'p Truffle::CExt.rb_ulong2num'
-e:1:in `method_missing': undefined method `rb_ulong2num' for Truffle::CExt:Module (NoMethodError)
Did you mean?  rb_long2int
               rb_num2ulong
               rb_num2long
               ULONG2NUM
	from -e:1:in `<main>'
```

Reply:
In `cext.rb` we can add:
```
def rb_ulong2num
   raise NotImplementedError, 'rb_ulong2num not implemented'
end
```

Reply:
That's less confusing than a NoMethodError, +1